etc/test-harness.lisp

```diff
@@ -5,16 +5,20 @@
 (in-package :test-harness)
 
 (defun test-system (system)
-  (handler-case 
+  (handler-case
       (ql:quickload system)
     (error (e)
       (format *standard-output*
               "~&Failed to Quickload dependencies for system `~a`:~&~a~&" system e)
       (uiop:quit -1)))
   (handler-case
       (if (asdf:test-system system)
-          (uiop:quit 0)
-          (uiop:quit -1))
+          (progn
+            (format *standard-output* "~&Test passed for system: `~a`~&" system)
```

Comment:
Strictly speaking, the semantic is that the test system invocation either succeeded or failure, not whether the tests passed.  Individual tests may fail during a `asdf:test-system` invocation while still returning a non-nil value. 

Better wording might be "Test system succeeded for system…" and "Test system failed for system…".

Reply:
It is "correct" but undesired.  We need to figure out how to get `lisp-unit` to return a non-zero result to the `asdf` machinery.

Reply:
From a look through the code for asdf-3.3.1, it unfortunately seems that `asdf:test-system` *always* returns `t` on invocation.  This seems to indicate that we can't get a non-zero status for failing tests running down this route.

Reply:
Time to fork `asdf`?